.changesets/don't-set-parameters-when-the-send_params-configuration-is-set-to-false.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "add"
```

Comment:
```suggestion
type: "fix"
```
Is it not a bug fix?

Reply:
No. This simply moves setting the params from the Plug integration (the only place params are set right now) to the main Elixir one. We’re doing this so we don’t have to implement it for the LiveView params again.